<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NetworkSettings</class>
 <widget class="QWidget" name="NetworkSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>345</width>
    <height>387</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="3" column="3">
      <widget class="QSpinBox" name="port">
       <property name="minimum">
        <number>1024</number>
       </property>
       <property name="maximum">
        <number>32767</number>
       </property>
       <property name="value">
        <number>30303</number>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLineEdit" name="listenIP">
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Automatic</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="4">
      <widget class="QCheckBox" name="hermitMode">
       <property name="text">
        <string>Hermit Mode</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="4">
      <widget class="QCheckBox" name="useUpnp">
       <property name="text">
        <string>Use UPnP</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2" colspan="2">
      <widget class="QLineEdit" name="clientName">
       <property name="placeholderText">
        <string>Anonymous</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2" colspan="2">
      <widget class="QLineEdit" name="forcePublicIP">
       <property name="placeholderText">
        <string>Automatic</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0" colspan="4">
      <widget class="QCheckBox" name="dropPeers">
       <property name="text">
        <string>Forget Peers from Last Session</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Ideal &amp;Peers</string>
       </property>
       <property name="buddy">
        <cstring>idealPeers</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Public IP</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>&amp;Listen on</string>
       </property>
       <property name="buddy">
        <cstring>port</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>&amp;Client Name</string>
       </property>
       <property name="buddy">
        <cstring>clientName</cstring>
       </property>
      </widget>
     </item>
     <item row="5" column="2" colspan="2">
      <widget class="QSpinBox" name="idealPeers">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="value">
        <number>5</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
